src/components/spreadsheet/utils/spreadsheet-gs-filter-utils.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { ExpertFilter } from '../../../services/study/filter';
+import yup from '../../utils/yup-config';
+import { FILTERS, ID, NAME, SPREADSHEET_GS_FILTER } from '../../utils/field-constants';
+
+export type ExpertFilterForm = Omit<ExpertFilter, 'type' | 'equipmentType' | 'topologyKind' | 'rules'>;
+
+export const spreadsheetGsFilterFormSchema = yup.object({
```

Comment:
That's a bit too much to define all this... Isn't it possible to have an item component that just returns a list of IDs and that we don't have too much boilerplate code to write to use it ?

Reply:
Ideally we should have a `DirectoryItemsInput` which isn't coupled with RHF, I haven't add it because I judged it out of scope for this issue but that would be a good time to implement it